SPECS/hyperv-daemons/CVE-2023-3269.nopatch

```diff
@@ -0,0 +1,6 @@
+CVE-2023-3269 - Introducing commit not present in lts
```

Comment:
Considering these are all nopatches, I think they don't need to go through fast-track and can go to `main` instead.

Reply:
Thanks. I was wondering about that and had asked elsewhere, but had yet to hear back. I'll switch the target.